cfme/control/explorer/policies.py

```diff
@@ -282,7 +283,7 @@ def update(self, updates):
         else:
             view.cancel_button.click()
         view = self.create_view(PolicyDetailsView, override=updates)
-        assert view.is_displayed
+        wait_for(lambda: view.is_displayed, timeout=10)
```

Comment:
Required: please add message for wait_for, this info what view we are waiting for